zkvm/tests/zkvm.rs

```diff
@@ -0,0 +1,72 @@
+use bulletproofs::r1cs;
+use bulletproofs::{BulletproofGens, PedersenGens};
+use curve25519_dalek::ristretto::CompressedRistretto;
+use curve25519_dalek::scalar::Scalar;
+use merlin::Transcript;
+use std::collections::VecDeque;
+
+use zkvm::*;
+
+fn issue_contract(qty: u64, flv: Scalar, pred: &Predicate) -> Vec<Instruction> {
+    vec![
+        // pushdata qty [qty]
+        Instruction::Push(Commitment::from(CommitmentWitness::from_secret(qty)).into()),
+        // stack: qty-var
+        Instruction::Var,
+        // stack: flv, qty-var
+        Instruction::Push(Commitment::from(CommitmentWitness::unblinded(flv)).into()),
+        // stack: flv-var, qty-var
+        Instruction::Var,
+        // stack: pred, flv-var, qty-var
+        Instruction::Push(pred.clone().into()),
+        // stack: issue-contract
+        Instruction::Issue,
+        // stack: pred, issue-contract
+        Instruction::Push(pred.clone().into()),
+        // stack: nonce-contract, issue-contract
+        Instruction::Nonce,
+        // stack: issue-contract
+        Instruction::Signtx,
+        // stack: issue-payload...
```

Comment:
"issue-payload" is always going to be the "issued-value", so we can document it as such explicitly